### PR TITLE
Fix bug by creating new method emptyFolderRecursive with exceptionPath

### PR DESCRIPTION
**Pull request by lfpvillegas**

### CODE REVIEW

File: packages/node-cli/src/services/FileSystemService.js [hunk 1]
```diff
@@ -110,6 +110,25 @@ module.exports = class FileSystemService {
 		}
 	}
 
+	emptyFolderRecursive(folderPath, exceptionPath) {
```
Comment (davidecorreu) [inline packages/node-cli/src/services/FileSystemService.js]: I will name the second parameter, excludePath.

Comment (lfpvillegas) [reply packages/node-cli/src/services/FileSystemService.js]: Done!

File: packages/node-cli/src/commands/CreateProjectCommandGenerator.js [hunk 2]
```diff
@@ -252,6 +252,9 @@ module.exports = class CreateProjectCommandGenerator extends BaseCommandGenerato
 
 		const actionCreateProject = new Promise(async (resolve, reject) => {
 			try {
+				if(answers[COMMAND_OPTIONS.OVERWRITE]) {
```
Comment (vic-nik) [inline packages/node-cli/src/commands/CreateProjectCommandGenerator.js]: formatting, space missing after if

File: packages/node-cli/src/services/FileSystemService.js [hunk 3]
```diff
@@ -110,6 +110,23 @@ module.exports = class FileSystemService {
...
 	}
 
+	emptyFolderRecursive(folderPath) {
+		assert(folderPath);
+		let self = this;
+		if (existsSync(folderPath)) {
+			readdirSync(folderPath).forEach(file => {
+				let currentPath = path.join(folderPath, file);
+				if (lstatSync(currentPath).isDirectory()) {
+					// recurse
```
Comment (vic-nik) [inline packages/node-cli/src/services/FileSystemService.js]: I would remove the comments, this code is readable

**Pull request merged**